docs/Catalog/Packager.md

```diff
@@ -0,0 +1,182 @@
+package name being the last two components of the source URI (e.g., `source/folder`).
+By default, it will assume the last component is a metadata file, and
+try to parse that as json.
+If you are specifying a folder, you should instead end the URI with a `/`.
+
+Optionally, you can control the package name, metadata, and other settings by
+explicitly specifying any of the following fields:
+
+```jsonc
+{
+  "source_prefix": "s3://data_bucket/source/folder/",
+  "registry": "package_bucket",  // may be the same as `data_bucket`
+  "package_name": "prefix/suffix",
+  "metadata": { "key": "value" },  // dictionary
+  "metadata_uri": "metadata.json", // alternative to `metadata`, relative or absolute
+  "message": "Commit message for the package revision", // string
+  "workflow": "alpha", // name of a valid metadata workflow
+  "should_copy": false // boolean
+  // false = point to data in the source bucket (default)
+  // true = copy data to the package bucket
+}
+```
+
+## Custom EventBridge Rules
+
+EventBridge event in your account (from any bus, in any region) into a
+conforming SQS message.
+
+### Example: Event-Driven Packaging (EDP)
+
+EDP is a high-end add-on to Quilt that coalesces multiple S3 uploads into a
+single `package-objects-ready` event, where it infers the appropriate top-level
+folder.  EDP writes to its own EventBridge bus, so you need to
+[Pipe](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes.html)
+it to the default bus to trigger the Auto-Packaging.
+
+When ready, it creates an event like this:
+
+```json
+{
+  "version":"0",
+  "id":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "detail-type":"package-objects-ready",
+  "source":"com.quiltdata.edp",
+  "account":"XXXXXXXXXXXX",
+  "time":"2022-12-08T20:01:34Z",
+  "region":"us-east-1",
+  "resources":[
+    "arn:aws:s3:::bucket-name"
+  ],
+  "detail":{
+    "version":"0.1",
+    "bucket":"bucket-name",
+    "prefix":"prefix-path-1/prefix-path-2/"
+  }
+}
+```
+
+Here is an example of an EventBridge rule that will use the above event
+to trigger packager queue:
+
+```json
+{
+  "EventPattern": {
+    "source": ["com.quiltdata.edp"],
+    "detail-type": ["package-objects-ready"]
+  },
+  "State": "ENABLED",
+  "Targets": [
+    {
+      "Id": "SQS_PackagerQueue",
+      "Arn": "arn:aws:sqs:us-east-1:XXXXXXXXXXXX:PackagerQueue-XXXXXXXXXXXX",
+      "InputTransformer": {
+        "InputPathsMap": {
+          "bucket": "$.detail.bucket",
+          "prefix": "$.detail.prefix"
+        },
+        "InputTemplate": "{ \"source_prefix\":\"s3://<bucket>/<prefix>/\" }"
+      }
+    }
+  ]
+}
+```
+
+## Caveats
+
+1. The package creation process is asynchronous, so you may need to wait a few
+   minutes before the package is available (longer if the source data is large).
+2. The SQS queue currently only accepts requests from EventBridge events via
+   custom rules.  Contact us at `support@quiltdata.io` if you have a use case
+   for which that is too restrictive.
```

Comment:
you can use `SendMessage` API if it's allowed by your identity-based permissions